collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -838,6 +866,136 @@ public void run() {
         refreshCurrentView();
     }
 
+    /**
+     * Save a copy of the chosen video in Collect's own path such as
+     * "/storage/emulated/0/odk/instances/{form name}/filename",
+     * and if it's from Google Drive and not cached yet, we'll retrieve it using network.
+     * This may take a long time.
+     *
+     * @param selectedVideo uri of the selected video
+     * @see #getFileExtensionFromUri(Uri)
+     */
+    private void saveChosenVideo(Uri selectedVideo) {
```

Comment:
`saveChosenVideo` and `saveChosenAudio` doing the same thing, so we can keep that as one single method.